transmogrifier/sources/json/aardvark.py

```diff
@@ -86,19 +88,106 @@ def get_optional_fields(self, source_record: dict) -> dict | None:
-        # related_items
+        # related_items not used in MITAardvark
 
         # rights
+        fields["rights"] = self.get_rights(source_record) or None
 
         # subjects
         fields["subjects"] = self.get_subjects(source_record) or None
 
         # summary field
+        fields["summary"] = source_record.get("dct_description_sm")
+
         return fields
 
+    @staticmethod
+    def get_alternate_titles(source_record: dict) -> list[timdex.AlternateTitle]:
+        """Get values from source record for TIMDEX alternate_titles field."""
+        alternate_titles = []
+
+        if "dct_alternative_sm" in source_record:
+            for title_value in [
+                title_value for title_value in source_record["dct_alternative_sm"]
+            ]:
+                alternate_titles.append(timdex.AlternateTitle(value=title_value))
+
+        return alternate_titles
+
+    @staticmethod
+    def get_contributors(source_record: dict) -> list[timdex.Contributor]:
+        """Get values from source record for TIMDEX contributors field."""
+        contributors = []
+
+        if "dct_creator_sm" in source_record:
+            for contributor_value in [
+                contributor_value
+                for contributor_value in source_record["dct_creator_sm"]
+            ]:
+                contributors.append(
+                    timdex.Contributor(value=contributor_value, kind="Creator")
+                )
+
+        return contributors
+
+    @staticmethod
+    def get_notes(source_record: dict) -> list[timdex.Note]:
+        """Get values from source record for TIMDEX notes field."""
+        notes = []
+
+        if "gbl_displayNote_sm" in source_record:
+            for note_value in [
+                note_value for note_value in source_record["gbl_displayNote_sm"]
+            ]:
+                notes.append(timdex.Note(value=[note_value], kind="Display note"))
+
+        return notes
+
+    @staticmethod
+    def get_publication_information(source_record: dict) -> list[str]:
+        """Get values from source record for TIMDEX publication_information field."""
+        publication_information = []
+
+        if "dct_publisher_sm" in source_record:
+            publication_information.extend(source_record["dct_publisher_sm"])
+
+        if "schema_provider_s" in source_record:
+            publication_information.append(source_record["schema_provider_s"])
+
+        return publication_information
+
+    @staticmethod
+    def get_rights(source_record: dict) -> list[timdex.Rights]:
+        """Get values from source record for TIMDEX rights field."""
+        rights = []
+
+        if "dct_accessRights_s" in source_record:
+            rights.append(
+                timdex.Rights(
+                    description=source_record["dct_accessRights_s"], kind="Access"
+                )
+            )
+
+        if "dct_license_sm" in source_record:
+            rights.append(timdex.Rights(uri=source_record["dct_license_sm"]))
```

Comment:
Because `dct_license_sm` has an `_sm` suffix, I think that means it's a list coming back.  Looking at the `Rights.uri` field, looks like it's expecting a string?  would this be problematic?